src/main/java/com/github/dockerjava/core/DockerClientConfig.java

```diff
@@ -102,8 +102,9 @@ private URI checkDockerHostScheme(URI dockerHost) {
     private String checkDockerCertPath(boolean dockerTlsVerify, String dockerCertPath) {
         if (dockerTlsVerify) {
             if (StringUtils.isEmpty(dockerCertPath)) {
-                throw new DockerClientException(
-                        "Enabled TLS verification (DOCKER_TLS_VERIFY=1) but certifate path (DOCKER_CERT_PATH) is not defined.");
+                return dockerCertPath;
+//                throw new DockerClientException(
```

Comment:
Hmm. I see... I don't like this. Maybe we have to rethink config class(es) again then. What about introducing a `DockerClientConfig` interface that must provide a `SSLContext`. Then we have a `DefaultDockerClientConfig` implementation of this `DockerClientConfig` interface that acts like the former `DockerClientConfig' class and someone (like you) could implement a second class that constructs the `SSLContext` in a different way (maybe from a database or whatever).